Rubberduck.Main/Root/RubberduckIoCInstaller.cs

```diff
@@ -269,7 +268,6 @@ private void RegisterParseTreeInspections(IWindsorContainer container, Assembly[
                 container.Register(Classes.FromAssembly(assembly)
                     .IncludeNonPublicTypes()
                     .Where(type => type.IsBasedOn(typeof(IParseTreeInspection)) && type.NotDisabledExperimental(_initialSettings))
-                    .WithService.Base()
                     .WithService.Select(new[] { typeof(IInspection) })
```

Comment:
This has to be extended by `typeof(IParseTreeInspection)` to account for the removed `Base()`.